release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -415,12 +415,12 @@ spec:
 [[ocp-311-scale]]
 === Scale
 
-[[ocp-311-scale-cluster-limits]]
-==== Cluster Limits
+[[ocp-311-scale-cluster-maximums]]
+==== Cluster Maximums
```

Comment:
In your other PR "Cluster Maximums" is not capitalized in section headings. Calling this out so you can standardize one way or the other. I'd go no caps.

Reply:
There are a handful of headings where I see this below as well.

Reply:
@lamek Thank you! We began applying sentence case headings in 4.x docs. This is for 3.11, so you'll see older standards applied.